nasa/earthdata-search
# EDSC-1603: added portal config for HiTIDE

# Overview

### What is the feature?

This adds a portal for [HiTIDE](https://hitide.podaac.earthdatacloud.nasa.gov/). 

### What is the Solution?

This adds the HiTIDE portal but modification to the config will be done in the future. Currently, the logo is PO.DAAC's until a HiTIDE logo is created and collections are filtered to be PO.DAAC collections. Further work will be done to filter the collections similar to what is in HiTIDE.

### What areas of the application does this impact?

Portals of Earthdata Search

# Testing

### Reproduction steps

- **Environment for testing: local**
- **Collection to test with: N/A**

1. Go to [http://localhost:8081/search?portal=hitide](http://localhost:8081/search?portal=hitide)
2. Check to make sure page loads properly and HiTIDE is correctly in the page portal title.

### Attachments

Please include relevant screenshots or files that would be helpful in reviewing and verifying this change.

# Checklist
